task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -9,7 +9,8 @@ metadata:
   name: sast-snyk-check
 spec:
   description: >-
-    Static application security testing with Snyk.
+    Uses Snyk Code tool to perform Static Application Security Testing (SAST) on application's source code.
```

Comment:
Suggestion based on how we have structured the descriptions above: Uses Snyk Code, a Static Application Security Testing (SAST) tool, to scan source code for security vulnerabilities, including common issues such as SQL injection, cross-site scripting (XSS), and code injection attacks.

Question: ....to scan source code.... Which source code? container?